tchannel/tornado/request.py

```diff
@@ -112,10 +114,10 @@ def get_header(self):
         :return: a future contains the deserialized value of header
         """
         raw_header = yield get_arg(self, 1)
-        if not self.scheme:
+        if not self.serializer:
```

Comment:
You gave it a default of `RawSerializer`. There won't be any case where `serializer` will be None -- unless the user makes it break manually, in which case this should blow up anyway.